HighAvailabilityModule.Algorithm/MembershipWithWitness.cs

```diff
@@ -11,6 +11,10 @@ public class MembershipWithWitness
     {
         private string Uuid { get; }
 
+        private string Utype { get; }
+
+        private string Unum { get; }
```

Comment:
Can we have a better name here? As the field will contain ip-address or machine name here.